wandb/sdk_py27/wandb_settings.py

```diff
@@ -152,6 +154,25 @@ def _get_program():
     return None
 
 
+# the setting exposed to users as `dir=` or `WANDB_DIR` is actually
+# the `root_dir`. We add the `__stage_dir__` to it to get the full
+# `wandb_dir`
+def get_wandb_dir(root_dir):
+    # We use the hidden version if it already exists, otherwise non-hidden.
+    if os.path.exists(os.path.join(root_dir, ".wandb")):
+        __stage_dir__ = ".wandb" + os.sep
```

Comment:
was this already a feature in the original cli?  The ".wandb" dir?

Reply:
Yeah -- I got this from the `wandb_dir()` function. I'm guessing it's a compatibility thing for old versions?